src/components/author.ts

```diff
@@ -50,7 +51,7 @@ const Author: FC<Props> = ({ first, id, image, last }) => {
           alignSelf: "flex-start",
         },
       },
-      h("span", null, first.toUpperCase()),
+      h("span", null, `${first.toUpperCase()} `),
```

Comment:
why this change?

Reply:
The separate lines are purely through styling, but if you copy+paste without the space then you'll paste their name without a space between first and last.
I should probably also be doing a `textTransform` in the style instead of a direct `toUpperCase`, that way it'll have the same case you want too.

Reply:
Huh, weirdly doing an `uppercase` in the textTransform will also copy it as all uppercased. Weird. No difference between the two then, I guess. I'd only bother taking it out if/when there's a proper JSS style for that component.